.storybook/config.js

```diff
@@ -25,12 +27,17 @@ function loadStories() {
 const context = require.context('../components', true, /\.twig$/)
 context.keys().forEach(key => {
   var template = context(key);
+  const rootPath = path.join(__dirname);
   Twig.twig({
+    base: rootPath, // Possibly can set a base directory for everything to work from?
+    namespaces: {
+      'atoms': 'components/01-atoms',
+    },
     id: key,
     data: template.tokens,
-    allowInlineIncludes: true,
+    // allowInlineIncludes: true, // This seems to be an alternative to namespacing which allows relative paths.
```

Comment:
I can't figure out if this is needed alongside namespaces or is an alternative. I think it is an alternative, but I can't find docs on it.